Makefile

```diff
@@ -757,7 +757,7 @@ eve-%: pkg/%/Dockerfile build-tools $(RESCAN_DEPS)
 	$(eval LINUXKIT_BUILD_PLATFORMS := --platforms $(subst $(space),$(comma),$(strip $(LINUXKIT_BUILD_PLATFORMS_LIST))))
 	$(eval LINUXKIT_FLAGS := $(if $(filter manifest,$(LINUXKIT_PKG_TARGET)),,$(FORCE_BUILD) $(LINUXKIT_DOCKER_LOAD) $(LINUXKIT_BUILD_PLATFORMS)))
 	$(QUIET)# ensures that we either pull it down or build it when we want to push
-	$(if $(filter push,$(LINUXKIT_PKG_TARGET)),$(QUIET)$(LINUXKIT) pkg build --pull $(LINUXKIT_BUILD_PLATFORMS) -build-yml $(call get_pkg_build_yml,$*) pkg/$*,)
+	$(if $(filter push,$(LINUXKIT_PKG_TARGET)),$(QUIET)$(LINUXKIT) pkg build --pull $(LINUXKIT_BUILD_PLATFORMS) $(LINUXKIT_DOCKER_LOAD) -build-yml $(call get_pkg_build_yml,$*) pkg/$*,)
```

Comment:
It is good question. Probably I will also understand it better if you will take a look onto https://github.com/lf-edge/eve/pull/2927#issuecomment-1315191041. Without the line you point onto I see:
```
$ rm -rf ~/.linuxkit/
$ ./build-tools/bin/linuxkit -v pkg push   --release snapshot --platforms linux/amd64 -build-yml build.yml pkg/mkconf
Building and pushing "lfedge/eve-mkconf:580c4b51ea2fc424fcb82536a7837e4dc060de4a"
checking for docker.io/lfedge/eve-mkconf:580c4b51ea2fc424fcb82536a7837e4dc060de4a in local cache...
docker.io/lfedge/eve-mkconf:580c4b51ea2fc424fcb82536a7837e4dc060de4a amd64 not found in local cache, checking registry
docker.io/lfedge/eve-mkconf:580c4b51ea2fc424fcb82536a7837e4dc060de4a amd64 found on registry
Pushing docker.io/lfedge/eve-mkconf:580c4b51ea2fc424fcb82536a7837e4dc060de4a
Error building and pushing "lfedge/eve-mkconf:580c4b51ea2fc424fcb82536a7837e4dc060de4a": could not find image or index for docker.io/lfedge/eve-mkconf:580c4b51ea2fc424fcb82536a7837e4dc060de4a
```
is it expected or I miss some flag?

Reply:
> It is good question

That is why I ask you, @giggsoff ; you often have good answers 😄 

> Without the line you point onto I see:

That is for `pkg/mkconf`, is it not? Rather than `pkg/eve`? Or are you saying the behaviour is the same?